### PR TITLE
Disabled all buttons/links that are not working right now

### PR DESCRIPTION
**Pull request by TSlashDreamy**
# **Disabled**
### GuestHomePage:
1. [NavBar] - Language icon
2. [Footer] - Term of use (Link)

### TutorPage/StudentPage:
1. [Navbar] - Language icon
2. [Navbar] - Message icon
3. [Navbar] - Saved icon
4. [Navbar] - Notifications icon

### MyProfile (Tutor):
1. [ProfileContainer] - Book a lesson button
2. [ProfileContainer] - Send message button
3. [ComentsBlock] - More reviews button

### MyCooperation:
1. [PageWrapper] - View my offers button

<details>
<summary>Preview</summary>
Disabled icon

Disabled icon

Disabled link
Disabled link

Disabled buttons
Disabled buttons

</details>

### CODE REVIEW

File: src/containers/tutor-profile/profile-info/ProfileInfo.jsx [hunk 1]
```diff
@@ -66,14 +66,16 @@ const ProfileInfo = () => {
   const buttonGroup = !isMyProfile && (
     <Box sx={styles.buttonGroup}>
       <Button
+        disabled
         fullWidth
         size={isDesktop ? SizeEnum.ExtraLarge : SizeEnum.Medium}
-        variant='containedLight'
+        variant='contained' // default: containedLight
```
Comment (Tolik170) [inline src/containers/tutor-profile/profile-info/ProfileInfo.jsx:72]: remove this comment or use containedLight without changes

File: src/containers/layout/footer/guest-footer/GuestFooter.tsx [hunk 2]
```diff
@@ -28,6 +28,7 @@ const GuestFooter = () => {
         </Typography>
         <Typography
           component={RouterLink}
+          sx={styles.disabledLink}
```
Comment (Tolik170) [inline src/containers/layout/footer/guest-footer/GuestFooter.tsx:31]: don`t think these styles make sense, you can remove the link or set '#'

**Review by maarekua — APPROVED** [review]

File: src/containers/layout/footer/guest-footer/GuestFooter.tsx (hunk 2 above)
Comment (TSlashDreamy) [reply src/containers/layout/footer/guest-footer/GuestFooter.tsx]: > don`t think these styles make sense, you can remove the link or set '#' or you can remove component link for now

you are right, the route is set to #

File: src/router/constants/guestRoutes.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ export const guestRoutes = {
   error: { route: 'error', path: 'error' },
   about: { route: 'about', path: 'about' },
   privacyPolicy: { route: 'privacy-policy', path: 'privacy-policy' },
-  termOfUse: { route: 'term-of-use', path: 'term-of-use' },
+  termOfUse: { route: '#', path: '#' },
```
Comment (MickhaelSh) [inline src/router/constants/guestRoutes.ts:9]: just remove, if its useless

**Review by MickhaelSh — APPROVED** [review]

**Pull request merged**